app/assets/stylesheets/modules/posts.scss

```diff
@@ -0,0 +1,354 @@
+.posts {
+  margin: 0 auto;
+  max-width: 960px;
+  text-align: left;
+  width: 70%;
+
+  &.for-archive {
+    box-sizing: border-box;
+    padding: 0 50px;
+
+    @include mq-mobile(){ padding: 0 20px; }
```

Comment:
@zalapeach I have seen different places where is missing a spaces after that the class/ids css declaration in this case is between the parenthesis "()" and the "{" :
```
@include mq-mobile(){ padding: 0 20px; }
# it might be
@include mq-mobile() { padding: 0 20px; }
```